test/Microsoft.AspNetCore.Routing.Tests/Constraints/DateTimeRouteConstraintTests.cs

```diff
@@ -36,9 +23,7 @@ public static IEnumerable<object[]> GetDateTimeObject
         [InlineData("12/25/2009 11:45:00 PM", true)]
         [InlineData("2009-05-12T11:45:00Z", true)]
         [InlineData("not-parseable-as-date", false)]
-        [InlineData(false, false)]
-        [MemberData(nameof(GetDateTimeObject))]
-        public void DateTimeRouteConstraint(object parameterValue, bool expected)
+        public void DateTimeRouteConstraint_ParsesStrings(string parameterValue, bool expected)
```

Comment:
@jbagga - this was the test that always got stuck. I think the problem was mixing `InlineData` and `MemberData`.